weblate_web/templates/snippets/reviews.html

```diff
@@ -0,0 +1,384 @@
+{% load i18n %}
+{% load static %}
+
+<style>
```

Comment:
Please remove the inline styles and move them to existing CSS files - weblate_web/static/style.css  and weblate_web/static/style-rtl.css for RTL variant.

Reply:
It has been moved.